docs/howto/openstack/nova/server-group.md

```diff
@@ -0,0 +1,41 @@
+# Using server groups
+
+Server groups provide a mechanism to group servers according to certain policy.
+
+In {{brand}} you can use server groups to control the scheduling of a group of servers. A server group can have one of four different policies; affinity, soft-affinity, anti-affinity or soft-anti-affinity.
+
+## Prerequisites
+
+In order to use server groups you must use the OpenStack CLI, go ahead and make sure you have [enabled it](/howto/getting-started/enable-openstack-cli).
+
+## Affinity
+
+A server group with the policy of "affinity" will make sure that all the servers in that group are **always** placed on the same physical compute node.
+
+While a policy of "soft-affinity" will **try to** make sure that all the servers in that group are placed on the same physical compute node, but ultimately will allow it if otherwise not possible.
+
+## Anti-affinity
+
+A server group with the policy of "anti-affinity" will make sure that all of the servers in that group are **always not** placed on the same physical compute node.
+
+> With "anti-affinity" you may also configure how many server you want to allow on the same physical compute node using, `--rule max_server_per_host=<number>`, where `<number>` is the amount of servers to allow on the same physical compute node.
+
+While a policy of "soft-anti-affinity", will **try to** make sure that all the servers in that group are not placed on the same physical compute node, but ultimately will allow it if otherwise not possible.
+
```

Comment:
This whole section should really be an Explanation. But since we don't have content in that section yet, I think we want to leave it in this guide now, and then move it as soon as we have some structure in the Explanation section.

Reply:
Yeah sounds reasonable, the different policies need to be explained somewhere.